### DISCUSSION TITLE
Enabling the Ctrl+A shortcut for beatmap listing

### DISCUSSION THREAD

**Discussion by ElenaVG0** [ppy/osu#17886] [Ideas]
When using Beatmap listing, not being able to use ctrl+a to quickly select and delete content in the search bar is a bit bothersome

**Comment by peppy** [comment]
It works here. Are you sure you haven't bound something else which is overriding it? What OS are you on?

**Reply by ElenaVG0** [reply]
I checked and have nothing bound to it, I'm on windows 10
